spec/comment_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'rails_helper'
+
+describe User, type: :model do
```

Comment:
- [ ] Kindly change this to Comment instead of User since that is what this file is all about.